### PR TITLE
Added datetime picker to sales & receivings forms (see #169)

### PR DESCRIPTION
**Pull request by daN4cat**
See discussion in #169 for more details.

### CODE REVIEW

File: application/views/partial/header.php [hunk 1]
```diff
@@ -32,7 +33,7 @@
 	<!-- end js template tags -->
     <?php else : ?>
     <!-- start minjs template tags -->
-    <script type="text/javascript" src="dist/opensourcepos.min.js?rel=3806e1c0ff" language="javascript"></script>
+    <script type="text/javascript" src="dist/opensourcepos.min.js?rel=c3fc0e5615" language="javascript"></script>
```
Comment (jekkos) [inline application/views/partial/header.php]: Looks like grunt cache breaker did run as the hash in the query string has changed?

Comment (daN4cat) [inline application/views/partial/header.php]: I run grunt that run your Gruntfile.js. 
I don't know the details of it, but I suppose that if you touch a file (having added a new jQuery plugin there is more now) an hashing needs to be updated. Right?

Comment (jekkos) [inline application/views/partial/header.php]: yes indeed, the fact that there's a new query string parameter present will assure that a user's cache will be refreshed (thanks to url rewriting). So a new hash means new minified javascript and a forced cache refresh.

Comment (daN4cat) [inline application/views/partial/header.php]: OK, so all good and makes sense.

Once you tested it a bit, please integrate and we close this a bit long story of broken plugin, new plugin and etc.

BTW, jQuery core is old and maybe should be updated. Any idea if it's a complicated matter? I'll create an issue for it later.

Comment (jekkos) [inline application/views/partial/header.php]: Yes I have looked into that beginning this year. At that time jQuery was a

**Comment by odiea** [comment]
Got a little carried away on that time picker did you?

**Comment by jekkos** [comment]
hehe yes.. I've added jquery ui sliders and now there's no more select boxes for that, this works out of the box.

**Comment by odiea** [comment]
Did you see js print setup has been updated in the last few months?

**Comment by daN4cat** [comment]
not sure about the slider thingy, would it work with a touch screen no problem?
That's why I avoided it.

**Comment by odiea** [comment]
Yes it works. I just tried it on my home screen and both the slider and dropdown in receivings work with a touch screen.

**Comment by daN4cat** [comment]
ok, good. Case closed then.

**Comment by daN4cat** [comment]
Almost closed... receiving is with the select not slider. Correct?

**Comment by odiea** [comment]
2015-10-18 11_55_15-once again thrift -- powered by os point of sale
Yes this is my image from report receiving

**Pull request merged**